eras/conway/impl/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 1.7.0.0
 
+* Revert `Cardano.Ledger.Conway.Gov` module name to `Cardano.Ledger.Conway.Governance`
```

Comment:
Instead of saying it was reverted it is better to remove the two entries below:
```
  * module `Cardano.Ledger.Conway.Governance` to `Cardano.Ledger.Conway.Gov`
  * module `Cardano.Ledger.Conway.Governance.Procedures` to `Cardano.Ledger.Conway.Gov.Procedures`
```
Since that change has never been released yet

Reply:
Yes I was wondering about that, but in the end I decided to treat the changelog as immutable hehe. 
But Ok, I'm fine with removing noise